tests/unit/models/test_peagle_data.py

```
@@ -0,0 +1,94 @@
+            loss_mask=_loss_mask(seq_len),
+            num_depths=4,
+        )
+        assert torch.equal(anchor_pos_none, anchor_pos_default)
+        assert torch.equal(depth_none, depth_default)
+
+    def test_max_anchors_fewer_valid_than_cap(self):
+        """When valid positions < max_anchors, all valid positions are used."""
+        seq_len = 16
+        loss_mask = torch.zeros(1, seq_len)
+        loss_mask[0, :5] = 1
+        anchor_pos, depth = generate_cod_sample_indices(
```

Ruff (0.15.20)

[warning] Unpacked variable `depth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)